alacritty_terminal/src/graphics/mod.rs

```diff
@@ -0,0 +1,146 @@
+//! This module implements the logic to manage graphic items included in a
+//! `Grid` instance.
+
+pub mod sixel;
+
+use std::mem;
+use std::sync::{Arc, Weak};
+
+use parking_lot::Mutex;
+use serde::{Deserialize, Serialize};
+
+use crate::term::color::Rgb;
+
+/// Max allowed dimensions (width, height) for the graphic, in pixels.
+pub const MAX_GRAPHIC_DIMENSIONS: (usize, usize) = (4096, 4096);
```

Comment:
If this were `[usize; 2]` instead, then in `fn graphics_attribute` you could write `&MAX_GRAPHIC_DIMENSIONS[..]`, which is much simpler than the `&[MAX_GRAPHIC_DIMENSIONS.0, MAX_GRAPHIC_DIMENSIONS.1][..]` you have there.

I’ve no idea if it would generate better code, but it’s certainly nicer to read.

Reply:
Modified in 3caa09ef9b339ebed1329eed016b8a5704bd2c77